src/app/modules/main/wallet_section/module.nim

```diff
@@ -94,31 +94,32 @@ proc newModule*(
   result.allTokensModule = all_tokens_module.newModule(result, events, tokenService, walletAccountService)
   result.collectiblesModule = collectibles_module.newModule(result, events, collectibleService, walletAccountService, networkService, nodeService, networkConnectionService)
   result.assetsModule = assets_module.newModule(result, events, walletAccountService, networkService, tokenService, currencyService)
-  result.transactionsModule = transactions_module.newModule(result, events, transactionService, walletAccountService, networkService, currencyService)
   result.sendModule = send_module.newModule(result, events, walletAccountService, networkService, currencyService, transactionService)
   result.savedAddressesModule = saved_addresses_module.newModule(result, events, savedAddressService)
   result.buySellCryptoModule = buy_sell_crypto_module.newModule(result, events, transactionService)
   result.overviewModule = overview_module.newModule(result, events, walletAccountService, currencyService)
   result.networksModule = networks_module.newModule(result, events, networkService, walletAccountService, settingsService)
   result.networksService = networkService
-  result.activityController = activityc.newController(result.transactionsModule, currencyService, tokenService, events)
+  result.transactionService = transactionService
+  result.activityController = activityc.newController(1, currencyService, tokenService, events)
```

Comment:
probably don't need to be used anywhere else, but it would be nice to have the requestIDs be an enum defined at the top (or some common place)